ui/mirage/scenarios/default.js

```diff
@@ -439,6 +440,168 @@ function policiesTestCluster(server) {
   server.createList('agent', 3, 'withConsulLink', 'withVaultLink');
 }
 
+function rolesTestCluster(server) {
```

Comment:
Our mocking library, Mirage, uses "scenarios" to scaffold clusters where you want to reuse the setup between tests. This adds a Roles test cluster scenario, like we'd previously done for variables, services, etc. tests. This (and anything else starting with /mirage) has no user-facing change.